webapp/components/_new/generic/UserAvatar/UserAvatar.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div :class="['user-avatar', size && `--${this.size}`]">
+    <span class="initials">{{ userInitials }}</span>
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T15:53:02Z" title="Wednesday, January 8th 2020, 4:53:02 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,106 @@
+<template>
+  <div :class="[`size-${this.size}`, 'user-avatar']">
+    <span v-if="!hasImage || error" class="no-image">
```


I'd still need to look at this in detail but my first guess is that we could get rid of the whole `error` handling and instead display the `anonymous` icon as placeholder until the image is loaded (if there even is one).